instructor/compiler.py

```diff
@@ -362,7 +363,7 @@ def micro_model(a):
 def generate_slides(assignment):
     ret = {}
     for image in assignment.image.all():
-        ret[image.id] = image.file.url
+        ret[image.id] = re.sub(r'\?[\S]+', '', image.file.url)
```

Comment:
I would use a `.split('?')[0]` rather than a regex here if the intent is just to remove the query parameters. To be more 'correct' I would use urllib, but that's a bit more cumbersome.